src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,9 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  user.name ? (
+    <a className="UserInfo" href={`mailto:${user.email}`}>
+      {user.name}
+    </a>
+  ) : ('Anonym')
```

Comment:
Refactor after check adding, no need to show 'Anonym'
```suggestion
export const UserInfo = ({ user }) => (
    <a className="UserInfo" href={`mailto:${user.email}`}>
      {user.name}
    </a>
```